### DISCUSSION TITLE
Installation of Roadrunner plugin for CARLA quickstart installation package

### DISCUSSION THREAD

**Discussion by 26Gpi98** [carla-simulator/carla#6608] [Q&A]
Hi guys, I wanted to ask if there are any possibilities to install "roadrunner" plugin for Quick start installation package of CARLA 0.9.13 ??
I want to import a xodr map file in CARLA.

I have read the documentation for plugin which suggest to download it from Mathworks website.

But, i figured that I need to build the CARLA and Unreal Engine from source. 

I'm I thinking correct about this? 
should i build carla and unreal engine from source? or there are any options available for Quickstart installation package to have roadrunner plugin.

**Comment by SExpert12** [comment]
I think you can also use packaged version. See this, specifically last lines:
https://carla.readthedocs.io/en/latest/tuto_M_generate_map/

**Reply by 26Gpi98** [reply]
yes, thank you for your reply.

**Reply by SExpert12** [reply]
Best wishes for your work.
